runtime/rasdump/javadump.cpp

```diff
@@ -993,6 +994,13 @@ JavaCoreDumpWriter::writeEnvironmentSection(void)
 	_OutputStream.writeCharacters(_VirtualMachine->memoryManagerFunctions->omrgc_get_version(_VirtualMachine->omrVM));
 	_OutputStream.writeCharacters("\n");
 
+#ifdef VENDOR_NAME
+	/* Write the VM version data */
+	_OutputStream.writeCharacters("1CI");
+	_OutputStream.writeCharacters(VENDOR_NAME,3);
```

Comment:
Please add a space after the comma.
I think we should expect `vendor_version.h` to also define `VENDOR_SHORT_NAME` (or something similar) which should be used here rather than the first three characters of `VENDOR_NAME`.